app/models/event.rb

```diff
@@ -147,7 +148,7 @@ def to_param
   # ActiveRecord :on_create
   def slugify!
     self.slug = "#{self.chapter.name} #{self.name.parameterize} #{self.id}".parameterize
-    self.save()
+    #self.save()
```

Comment:
Why?

Reply:
I had to explicitly called event.save in https://github.com/null-open-security-community/swachalit/blob/5a06db4d8be4bfd9dcfb17beb759965445b364e1/db/seeds.rb#L112 and wanted to have slug to be created before.